loops_lab.js

```diff
@@ -0,0 +1,230 @@
+// ## Exercises
+//
+// 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
+
+let numA = 100;
```

Comment:
Good demonstration of for and while loops. No major feedback here. I think you understand the usage and implementation of loops. Keep up the good work!

Reply:
Thanks JR !